src/ga_cache.cpp

```diff
@@ -29,6 +29,9 @@ namespace sdk {
 
         static auto get_db()
         {
+            // Verify thread safety in the event that sqlite has been upgraded
+            GDK_RUNTIME_ASSERT(sqlite3_threadsafe() != SQLITE_CONFIG_SINGLETHREAD);
```

Comment:
It seems `sqlite3_threadsafe()` returns the value of `SQLITE_THREADSAFE` (0, 1, 2) and not:

```
#define SQLITE_CONFIG_SINGLETHREAD  1  /* nil */
#define SQLITE_CONFIG_MULTITHREAD   2  /* nil */
#define SQLITE_CONFIG_SERIALIZED    3  /* nil */
```

Maybe this should be

```suggestion
            GDK_RUNTIME_ASSERT(sqlite3_threadsafe());
```

Reply:
Updated, thanks!